src/Module/Api/Mastodon/Accounts/Search.php

```diff
@@ -51,19 +51,11 @@ protected function rawContent(array $request = [])
 
 		$accounts = [];
 
-		if (!$request['following']) {
-			if ((strrpos($request['q'], '@') > 0) && $request['resolve']) {
+		if ($request['resolve']) {
+			if ((strrpos($request['q'], '@') > 0)) {
```

Comment:
Superfluous parentheses.
```suggestion
			if (strrpos($request['q'], '@') > 0) {
```